### ISSUE TITLE
SVG map support

### ISSUE THREAD

**Issue by ivanzhujunwei** [kartograph/kartograph.org#75]
Hi,

Do I have any choice to use other SVG maps?
I found a website (https://www.amcharts.com/svg-maps/) providing many SVG maps, but there is a problem when I use them. I tried other svg maps, still not working. 
![wx20170521-122411](https://cloud.githubusercontent.com/assets/14146339/26280653/de263588-3e20-11e7-9ff3-953597816a14.png)
Thanks!

**Comment by x4dr** [comment]
yeah i tried to use it with https://upload.wikimedia.org/wikipedia/commons/5/56/Berlin_Postleitzahlen.svg but apparently its the wrong kind of svg map ... maybe stick a big warning that the svgs HAVE to be created with kartograph.py?
